src/backend/marsha/core/tests/test_utils_medialive_utils.py

```diff
@@ -789,3 +789,35 @@ def test_delete_aws_elemental_stack(self):
             )
 
             medialive_utils.delete_aws_element_stack(video)
+
+    def test_delete_mediapackage_channel(self):
+        """Shoult delete a mediapackage channel and its endpoint."""
```

Comment:
```suggestion
        """Should delete a mediapackage channel and its endpoint."""
```

Reply:
This code does not exists anymore, the same function was introduced in #1083